src/main/java/org/folio/marc4ld/service/marc2ld/mapper/mapper/GovernmentPublicationMapper.java

```diff
@@ -57,7 +56,7 @@ public void map(MarcData marcData, Resource resource) {
   private void processControlField(Resource resource, ControlField controlField) {
     var code = controlField.getData().charAt(GOVT_PUB_CHAR_INDEX);
     var link = LINK_MAPPER.apply(code);
-    var term = MARC_CODE_TO_TERM_MAP.getOrDefault(code, EMPTY);
```

Comment:
these changes are redundant (same in ```TargetAudienceMapper```)
currently there is a bug in ```FieldControllerImpl``` which I have fixed in this PR (in short, the ```marc2ldCondition``` of edge resources is being completely ignored)